Measurement/twinkles_light_curves.ipynb

```diff
@@ -0,0 +1,480 @@
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "fig = plt.figure(figsize=(12,8))\n",
+    "\n",
+    "display1 = afw_display.Display(frame=1, backend='matplotlib')\n",
+    "display1.scale('asinh', 'zscale')\n",
+    "display1.mtv(calexp.image)\n",
+    "display1.zoom(80)\n",
+    "display1.pan(x_lens, y_lens)\n",
+    "plt.scatter(x_lens, y_lens, c='r', marker='+', label='Lens', s=400, lw=2)\n",
+    "plt.scatter(x_image_1, y_image_1, c='b', marker='+', label='Lensed AGN Image 1', s=400, lw=2)\n",
+    "plt.scatter(x_image_2, y_image_2, c='g', marker='+', label='Lensed AGN Image 2', s=400, lw=2)\n",
+    "plt.legend(fontsize=32)\n",
+    "plt.xlabel('X Pixels', size=28)\n",
+    "plt.ylabel('Y Pixels', size=28)\n",
+    "plt.xticks(size=18)\n",
+    "plt.yticks(size=18)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Get Source Catalogs and match to Truth\n",
+    "\n",
+    "Here we'll use the butler to get source catalogs for all the visits in the `r` band and match these source catalogs to our truth. First we use the butler to get the subset of r-band visits."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "r_band_subset = butler.subset('src', filter='r')"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Let's take the source catalog from the first visit and make the same plot as before but this time include all identified sources in the DM catalog."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "src_cat = butler.get('src', dataId = r_band_subset.cache[0]).asAstropy().to_pandas()"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "src_pix = [wcs.skyToPixel(SpherePoint(Angle(float(src_cat['coord_ra'][idx])),\n",
+    "                                      Angle(float(src_cat['coord_dec'][idx]))))for idx in range(len(src_cat))]"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "fig = plt.figure(figsize=(12,8))\n",
+    "\n",
+    "display1 = afw_display.Display(frame=1, backend='matplotlib')\n",
+    "display1.scale('asinh', 'zscale')\n",
+    "display1.mtv(calexp.image)\n",
+    "display1.zoom(80)\n",
+    "display1.pan(x_lens, y_lens)\n",
+    "plt.scatter(x_lens, y_lens, c='r', marker='+', label='Lens', s=400, lw=2, zorder=10)\n",
+    "plt.scatter(x_image_1, y_image_1, c='b', marker='+', label='Lensed AGN Image 1', s=400, lw=2, zorder=10)\n",
+    "plt.scatter(x_image_2, y_image_2, c='g', marker='+', label='Lensed AGN Image 2', s=400, lw=2, zorder=10)\n",
+    "for idx in range(len(src_pix)):\n",
+    "    src_loc = src_pix[idx]\n",
+    "    plt.scatter(src_loc[0], src_loc[1], c='orange', s=100, zorder=1)\n",
```

Comment:
You can also use `afwDisplay.dot` to put markers down on the images, I think.

That doesn't make sense if you want to use the vector version of scatter, but when looping over a list, I think it's more idiomatic to use `dot`.